lib/kbuildbarn/exec/bb_clientd.go

```diff
@@ -75,6 +75,25 @@ func MaybeStartClient(o *ClientOptions) (*Client, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error starting bb_clientd: %w", err)
 	}
+
+	// Wait for a directory under the mount dir to appear, so that we know
+	// bb_clientd is ready, up until `timeout`
+	stopTime := time.Now().Add(timeout)
```

Comment:
retry lib? 5 times retry, 100 ms in between? not sure it's a good idea, but would save some typing here.

Reply:
Retry lib seems to work great! Thanks for the pointer